### ISSUE TITLE
Blank submitted answer is confusing for pl-file-editor

### ISSUE THREAD

**Issue by BenjaminCosman** [PrairieLearn/PrairieLearn#9806]
<img width="840" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/4876098/cc92db5c-306f-4fb8-87f0-d152b762363a">

As shown above, when you click Save on a pl-file-editor element, the "Submitted answer" at the bottom of the page is blank. I know from a handful of emails and CBTF incident reports that students sometimes believe this means their answer did not save correctly. Perhaps instead, either the box should show 1) the text of the saved answer, 2) the "Preview" of the saved answer, or 3) a message like "Your answer was saved."

**Comment by jonatanschroeder** [comment]
For pl-file-editor the typical use is that you can add to the bottom of your question.html:

```
<pl-submission-panel>
  <pl-file-preview></pl-file-preview>
</pl-submission-panel>
```

**Comment by jonatanschroeder** [comment]
If you want to see a formatted preview of your markdown submission instead, you can use:

```html
<pl-submission-panel>
   <pl-xss-safe submitted-file-name="set1.md" language="markdown"></pl-xss-safe>
</pl-submission-panel>
```

**Comment by BenjaminCosman** [comment]
I'll do that in the future; thanks! For folks who do not use this though, I wonder if one of the three options above might be a better default than a blank box? Most other elements show some representation of the submitted answer by default rather than a blank box.

**Comment by jonatanschroeder** [comment]
@mwest1066 can probably talk to this further, but I believe changing this now may affect a lot of courses that rely on the current behavior.

**Comment by mwest1066** [comment]
I agree that the current behavior is not good, and that we really should have some kind of default submission display. The backward compatibility problem that @jonatanschroeder mentions is definitely a problem, however. @nwalters512 tagging you in on this in case you've got any good ideas.
